diff --git a/TestApps/MessagingDemoAppSwiftUI/AppPages/ElementViews/TabHeader.swift b/TestApps/MessagingDemoAppSwiftUI/AppPages/ElementViews/TabHeader.swift
index bf6abbc..71517b2 100644
--- a/TestApps/MessagingDemoAppSwiftUI/AppPages/ElementViews/TabHeader.swift
+++ b/TestApps/MessagingDemoAppSwiftUI/AppPages/ElementViews/TabHeader.swift
@@ -16,7 +16,7 @@ struct TabHeader: View {
     var title: String
     var refreshAction: (() -> Void)?
     var redownloadAction: (() -> Void)?
-    
+
     var body: some View {
         ZStack {
             Text("\(title)")